aws-observability/templates/test/TestTemplate.yaml

```diff
@@ -1694,30 +1703,32 @@ Tests:
             Section2aInstallApp: 'No'
             Section3aNLBLogsSourceName: ''
             Section3bNLBSourceUpdated: 'No'
+
   - TestName: existingsourcewithalbbucket
     Regions:
-      - us-east-1
+      - ap-southeast-1
     Skip: false
     Parameters:
       Values:
-        Section1aSumoLogicDeployment: 'us1'
-        Section1bSumoLogicAccessID: '${ENTERPRISE_US1_ACCESS_ID}'
-        Section1cSumoLogicAccessKey: '${ENTERPRISE_US1_ACCESS_KEY}'
-        Section1dSumoLogicOrganizationId: '${US1_ENTERPRISE_ORG_ID}'
+        Section1aSumoLogicDeployment: 'us2'
+        Section1bSumoLogicAccessID: '${FREE_ACCOUNT_ACCESS_ID}'
+        Section1cSumoLogicAccessKey: '${FREE_ACCOUNT_ACCESS_KEY}'
+        Section1dSumoLogicOrganizationId: '${US2_FREE_ORG_ID}'
         Section2aAccountAlias: 'test5env'
         Section2bAccountAliasMappingS3URL: ''
         Section3aInstallObservabilityApps: 'No'
         Section4aCreateMetricsSourceOptions: 'None'
-        Section4cCloudWatchExistingSourceAPIUrl: 'https://api.us2.sumologic.com/api/v1/collectors/167776029/sources/841621157'
+        Section4cCloudWatchExistingSourceAPIUrl: 'https://api.us2.sumologic.com/api/v1/collectors/350831942/sources/2763251087'
         Section5aAutoEnableS3LogsALBResourcesOptions: 'Existing'
         Section5bALBCreateLogSource: 'Yes'
-        Section5dALBS3LogsBucketName: 'cf-templates-1qpf3unpuo1hw-us-east-1-arun'
+        Section5dALBS3LogsBucketName: 'cf-templates-1qpf3unpuo1hw-ap-southeast-1'
         Section5eALBS3BucketPathExpression: '*abc*'
         Section6aCreateCloudTrailLogSource: 'Yes'
         Section7aLambdaCreateCloudWatchLogsSourceOptions: 'None'
-        Section7bLambdaCloudWatchLogsSourceUrl: 'https://api.us2.sumologic.com/api/v1/collectors/167776029/sources/1169994000'
+        Section7bLambdaCloudWatchLogsSourceUrl: 'https://api.us2.sumologic.com/api/v1/collectors/350831942/sources/2763257213'
         Section7cAutoSubscribeLogGroupsOptions: 'New'
         Section8aRootCauseExplorerOptions: 'Xray Source'
+        Section9bELBCreateLogSource: 'No'
```

Comment:
Why create El resource is set to. No

Reply:
This is `ALB` testcase, we don't required ELB creation.